Tools/setup/ubuntu.sh

```diff
@@ -263,15 +317,50 @@ if [[ $INSTALL_SIM == "true" ]]; then
 		pkg-config \
 		protobuf-compiler \
 		;
+fi
 
-	if sudo dmidecode -t system | grep -q "Manufacturer: VMware, Inc." ; then
-		# fix VMWare 3D graphics acceleration for gazebo
-		echo "export SVGA_VGPU10=0" >> ~/.profile
-	fi
+# New Gazebo
+if [[ $INSTALL_GAZEBO == "true" ]]; then
+
+	echo
+	echo $VERBOSE_BAR
+	echo "🍻 Installing Gazebo"
+	echo
+	echo "  * Gazebo Garden"
+	echo $VERBOSE_BAR
+
+	install_gazebo_common
```

Comment:
`install_gazebo_common` is called twice. The first one is after `if [[ $INSTALL_SIM == "true" ]]; then`